src/utils/helper/index.js

```diff
@@ -0,0 +1,22 @@
+export const generateTaskObject = (title) => {
+  return {
+    id: Date.now().toString(),
+    title: title,
+    createdAt: new Date(),
+    isCompleted: false,
+    completedAt: null,
+  };
+};
+
+export const formatDate = (date) => {
+  const today = new Date(date);
```

Comment:
it `date` passed as a string or `Date` type ? possible [re-conversion](https://github.com/azo-wtag/todo-app-react/pull/1/files#diff-aa0f42ae78187eddfdde29478cb9c548c91812165bacd73f576adcef1374f51aR38) of a Date.